client/src/rtc/index.js

```diff
@@ -48,7 +48,7 @@ const manager = {
     connection.close();
   },
 
-  peerSocket: () => connection.socket,
+  id: () => connection.id()
```

Comment:
It was added because Room component needs to know the socket used to set up its peer connections. Before, the socket was being exported, but now it's the id